README.md

```diff
@@ -155,9 +159,13 @@ connect to empd (port 4369) on <hostname>." Note: RabbitMQ startup error would s
 and not in RabbitMQ logs (/var/log/rabbitmq/rabbitmq@hostname.log)
 
 ##### 3. Bootstrap
+Remove older version of rabbitmq_server directory. Defaults to <user_home>/rabbitmq_server/rabbitmq_server-3.9.7
```

Comment:
If updating remove existing?  directory?

Reply:
We no longer use that directory. Are you asking if bootstrap process will remove it?

Reply:
You assume that they  are updating.

Reply:
Updated wording